bzt/modules/reporting.py

```diff
@@ -572,7 +576,10 @@ def report_test_case(self, suite_name, case_name, children=None):
             system_out = etree.Element("system-out")
             system_out.text = "".join(self.report_urls)
             children.insert(0, system_out)
-        self.add_test_case(suite_name, attributes={"classname": self.class_name, "name": case_name}, children=children)
+        if duration is not None:
+            self.add_test_case(suite_name, attributes={"classname": self.class_name, "name": case_name, "time": duration}, children=children)
```

Comment:
Is it possible to pass default value to attributes without any erros?

Reply:
@Merilian , I'm not sure if I've understood your question. What I added here is the availability to specify the duration of the test using the "time" attribute on the <testcase> element.
If duration argument is not specified on the report_test_case() function, then the existing behavior is preserved.